src/Criteria/Criteria.php

```diff
@@ -49,10 +49,29 @@ public function toQuery(): string
         return $sql;
     }
 
+    public function toQuerySearch(): string
+    {
+        $sql = 'WHERE ';
+        $sql .= implode(' AND ', array_map(function ($filterName) {
+            return sprintf('%s LIKE %s', $filterName, ':' . $filterName);
+        }, array_keys($this->filters), $this->filters));
```

Comment:
las argument (`$this->filters`) is not needed, since you are only using the key in the callback